sympy/geometry/tests/test_line.py

```diff
@@ -54,13 +54,13 @@ def test_line_geom():
     raises(ValueError, lambda: Line(Point(0, 0), Point(0, 0)))
 
     # Basic stuff
-    assert Line((1, 1), slope=1) == Line((1, 1), (2, 2))
```

Comment:
My concern here is that the use of `==` affirms the form/arguments the result for a Line instantiated with a keyword should have. The use of `equals` would permit a divergence from that. Perhaps (if `equals` is not tested well) a few tests to improve coverage alone would be a better improvement to the testing file.